project/AkkaBuild.scala

```diff
@@ -28,7 +28,7 @@ object AkkaBuild {
     Formatting.formatSettings ++
     Protobuf.settings ++ Seq(
       parallelExecution in GlobalScope := System.getProperty("akka.parallelExecution", parallelExecutionByDefault.toString).toBoolean,
-      version in ThisBuild := "2.5-SNAPSHOT"
+      version in ThisBuild := "2.5.16"
```

Comment:
Please revert this. Akka changes -SNAPSHOT things during releasing with scripts; this should not be changed like this in-place